x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -139,7 +148,7 @@ export class TelemetryEventsSender {
   }
 
   public async fetchEndpointMetrics(executeFrom: string, executeTo: string) {
-    if (this.esClient === undefined) {
+    if (this.esClient === undefined || this.esClient === null) {
```

Comment:
not necessary since I think being more explicit is clearer, but for your personal knowledge you could just do `if(this.esClient == null)` for reference: http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript.html